cpu/saml21/startup.c

```diff
@@ -79,102 +80,80 @@ void reset_handler(void)
-void isr_tc3(void)                  __attribute__ ((weak, alias("dummy_handler")));
-void isr_tc4(void)                  __attribute__ ((weak, alias("dummy_handler")));
-void isr_adc(void)                  __attribute__ ((weak, alias("dummy_handler")));
-void isr_ac(void)                   __attribute__ ((weak, alias("dummy_handler")));
-void isr_dac(void)                  __attribute__ ((weak, alias("dummy_handler")));
-void isr_ptc(void)                  __attribute__ ((weak, alias("dummy_handler")));
-void isr_aes(void)                  __attribute__ ((weak, alias("dummy_handler")));
-void isr_trng(void)                 __attribute__ ((weak, alias("dummy_handler")));
+void isr_pm(void)               __attribute__ ((weak, alias("dummy_handler")));
+void isr_wdt(void)              __attribute__ ((weak, alias("dummy_handler")));
+void isr_rtc(void)              __attribute__ ((weak, alias("dummy_handler")));
+void isr_eic(void)              __attribute__ ((weak, alias("dummy_handler")));
+void isr_nvmctrl(void)          __attribute__ ((weak, alias("dummy_handler")));
+void isr_dmac(void)             __attribute__ ((weak, alias("dummy_handler")));
+void isr_usb(void)              __attribute__ ((weak, alias("dummy_handler")));
+void isr_evsys(void)            __attribute__ ((weak, alias("dummy_handler")));
+void isr_sercom0(void)          __attribute__ ((weak, alias("dummy_handler")));
+void isr_sercom1(void)          __attribute__ ((weak, alias("dummy_handler")));
+void isr_sercom2(void)          __attribute__ ((weak, alias("dummy_handler")));
+void isr_sercom3(void)          __attribute__ ((weak, alias("dummy_handler")));
+void isr_sercom4(void)          __attribute__ ((weak, alias("dummy_handler")));
+void isr_sercom5(void)          __attribute__ ((weak, alias("dummy_handler")));
+void isr_tcc0(void)             __attribute__ ((weak, alias("dummy_handler")));
+void isr_tcc1(void)             __attribute__ ((weak, alias("dummy_handler")));
+void isr_tcc2(void)             __attribute__ ((weak, alias("dummy_handler")));
+void isr_tc0(void)              __attribute__ ((weak, alias("dummy_handler")));
+void isr_tc1(void)              __attribute__ ((weak, alias("dummy_handler")));
+void isr_tc2(void)              __attribute__ ((weak, alias("dummy_handler")));
+void isr_tc3(void)              __attribute__ ((weak, alias("dummy_handler")));
+void isr_tc4(void)              __attribute__ ((weak, alias("dummy_handler")));
+void isr_adc(void)              __attribute__ ((weak, alias("dummy_handler")));
+void isr_ac(void)               __attribute__ ((weak, alias("dummy_handler")));
+void isr_dac(void)              __attribute__ ((weak, alias("dummy_handler")));
+void isr_ptc(void)              __attribute__ ((weak, alias("dummy_handler")));
+void isr_aes(void)              __attribute__ ((weak, alias("dummy_handler")));
+void isr_trng(void)             __attribute__ ((weak, alias("dummy_handler")));
 
```

Comment:
**ck the line counts, but when I see something that is keeping to the 80 char line length and it is soo easy to fix I can just not ignore it...